static/app/components/editableText.tsx

```diff
@@ -151,6 +153,7 @@ function EditableText({
             value={inputValue}
             onChange={handleInputChange}
             onFocus={event => autoSelect && event.target.select()}
+            maxLength={maxLength}
```

Comment:
I wonder if we should change this component so it just spreads props into the the Input?

Reply:
I think since we don't pass many props down, making it explicit would be preferable, but no strong opinions here :)

Reply:
I agree with Priscila, I'd prefer to make it explicit instead of spreading onto the input because there aren't many other needs for it now.

If we did want to do it later, I imagine we'd have to add an `inputProps` prop because there's also a label that renders when not editing so we can separate what goes where